types/strategy.ts

```diff
@@ -12,3 +12,23 @@ export interface Strategy {
   executePlugin: string;
   managePlugin: string;
 }
+
+export const strategySchema: Readonly<{
```

Comment:
This is not a type, and should not be added here. You can put it in utils, validations, etc.